src/http.js

```diff
@@ -91,29 +137,25 @@ Zotero.HTTP = new function() {
 			cookieSandbox: request.jar(),
 			debug: false,
 			logBodyLength: 1024,
-			timeout: 15000,
+			timeout: 90000,
```

Comment:
I guess this value should be the same for the timeout here and the default domain timeouts used earlier, so it might be better to either define a variable and re-use that, or get it from the configuration file (similar to line 147, `'User-Agent': config.get('userAgent')`)

Reply:
Different concepts. Renamed variables to make it more obvious.